CONTRIBUTIONS.md

```diff
@@ -0,0 +1,87 @@
+# 🧑‍💼Contributing to Deakin Detonator Toolkit
+
+Thank you for your interest in contributing to the Deakin Detonator Toolkit aka PT-GUI project! This project is student run and we welcome contributions from the students of Deakin to enhance this penetration testing toolkit. Please follow the guidelines below to ensure a smooth contribution process.
+
+## Table of Contents
+
+-   [Getting Started](#getting-started)
+-   [How to Contribute](#how-to-contribute)
+-   [Coding Standards](#coding-standards)
+-   [Submitting Contributions](#submitting-contributions)
+-   [Reporting Issues](#reporting-issues)
+
+## 👶Getting Started
+
+To get started with contributing, follow these steps; If you're not familiar with using Git team recommends you to first understand how the **[Git](https://deakin365.sharepoint.com/:w:/s/HardhatEnterprises2/Ee1Nlb_OZDVIlElD5RBO5o0BzG0OTRTFHIhWpTNwaxthmQ?e=Bg0rTb)** works with **[GitHub](https://deakin365.sharepoint.com/:b:/s/HardhatEnterprises2/ETU_JVpc67BCqgp6W7vm9ewBzbWCnlneQc-mzhiOIwGd8g?e=2pW0ld)**.
```

Comment:
Please change "If you're not familiar with using Git team recommends you to first understand how the..." to "If you're not familiar with using Git, we recommend you to first understand how...", as this reads a lot better.

Reply:
Hi Cam, I have made the necessary changes, requesting you to have a look at it.